### PR TITLE
Migrate to StyledSelect on Discover page

### PR DESCRIPTION
**Pull request by Nelson-Chinedu**
migrate from DeprecatedStyledSelect to StyledSelect component on discover page

### CODE REVIEW

**Comment by Nelson-Chinedu** [comment]
for the value of the width to change is it inside the ```<StyledSelect>``` component as a property because I could not find ```width``` as a property

**Comment by Betree** [comment]
@Nelson-Chinedu Please use `minWidth`. `StyledSelect` uses the `space` helper from `styled-system`: https://styled-system.com/api/#layout

**Comment by Nelson-Chinedu** [comment]
@Betree do you mean something like what i did for the changes

File: pages/discover.js [hunk 1]
```diff
@@ -251,6 +252,7 @@ const DiscoverPage = ({ router, intl }) => {
                           setRouteParam('sort', selected && selected.key);
                         }
                       }}
+                      minwidth={space}
```
Comment (Betree) [inline pages/discover.js]: Please see https://styled-system.com/api/#layout, this is not the correct spelling for this prop.

Also can you add screenshots of the changes to the pull request?

Comment (Nelson-Chinedu) [reply pages/discover.js]: Ok. will work to that

**Pull request closed without merging**